x-pack/plugins/index_management/public/application/components/mappings_editor/constants/data_types_definition.tsx

```diff
@@ -851,6 +851,23 @@ export const TYPE_DEFINITION: { [key in DataType]: DataTypeDefinition } = {
       </p>
     ),
   },
+  sparse_vector: {
```

Comment:
Hi @carlosdelest, just a question: why are we adding these changes to the Index management plugin?
Did you mean to add support for the `sparse_vector` field type in the Mappings editor?

<img width="1889" alt="Screenshot 2023-10-06 at 17 52 47" src="https://github.com/elastic/kibana/assets/59341489/30737d55-9062-423b-aedb-58d233fd0991">

Reply:
@ElenaStoeva , thanks for pointing that out - I totally missed it.

I've made the changes [here](https://github.com/elastic/kibana/pull/168186/commits/0a29bfe5cdeb06bab9c52a0e628fccad12bed8a5).

![image](https://github.com/elastic/kibana/assets/6339205/c3266335-1424-405a-9818-3f28f4a33dd5)

Reply:
Thanks for clarifying! Tested locally and LGTM.